content/home/getting-started.md

```diff
@@ -36,7 +36,7 @@ func main() {
 Cobra provides its own program that will create your application and add any
 commands you want. It's the easiest way to incorporate Cobra into your application.
 
-[Here](https://github.com/spf13/cobra/blob/master/cobra/README.md) you can find more information about it.
+[Here](https://github.com/spf13/cobra/blob/master/README.md) you can find more information about it.
```

Comment:
I think this is the link we want: 
https://github.com/spf13/cobra-cli/blob/main/README.md

Reply:
Although I just realised this PR is a duplicate of #3, which also hasn't been reviewed.

Reply:
Okk if it is already there,  then you can close this one